src/app/cli/src/commands/export_command.rs

```diff
@@ -0,0 +1,99 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use std::path::PathBuf;
+use std::sync::Arc;
+
+use kamu::domain::{ExportFormat, ExportService, QueryService};
+use opendatafabric::DatasetRef;
+
+use crate::{CLIError, Command};
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+pub struct ExportCommand {
+    export_service: Arc<dyn ExportService>,
+    query_service: Arc<dyn QueryService>,
+    dataset_ref: DatasetRef,
+    output_path: Option<PathBuf>,
+    output_format: String,
```

Comment:
Don't we have enums like the following in other CLI commands:

```
#[derive(Debug, Clone, Copy, clap::ValueEnum)]
pub enum OutputFormat {
    /// Comma-separated values
    #[clap(name = "csv")]
    Csv,

    /// Array of Structures format
    #[clap(name = "json")]
    Json,

    /// One Json object per line - easily splittable format
    #[clap(name = "ndjson")]
    NdJson,

    /// Structure of arrays - more compact and efficient format for encoding
    /// entire dataframe
    #[clap(name = "json-soa")]
    JsonSoA,

    /// Array of arrays - compact and efficient and preserves column order
    #[clap(name = "json-aoa")]
    JsonAoA,

    /// A pretty human-readable table
    #[clap(name = "table")]
    Table,
}
```

Reply:
I don't mean using this particular one, but perhaps your own `ExportFormat` listing valid options.